src/coreclr/debug/createdump/specialdiaginfo.h

```diff
@@ -0,0 +1,32 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+// ******************************************************************************
+// WARNING!!!: This code is also used by SOS in the diagnostics repo. Should be
+// updated in a backwards and forwards compatible way.
+// See: https://github.com/dotnet/diagnostics/blob/main/src/SOS/inc/specialdiaginfo.h
```

Comment:
This URL appears to be invalid.

Reply:
I haven't got the diagnostics side of these changes in yet.  That is a placeholder until I push the diag side of these changes.